index.html

```diff
@@ -1,11 +1,17 @@
 <DOCTYPE html>
   <html>
-    <head>
       <title>Touchcreator's site</title>
-      <h1>touchcreator.github.io</h1>
-<p>This is my website. So touchcreator.tk is old now. This also means that touchcreator.tk will now redirect here. this site is far from finished. </p>
-      </head>
-    <body>
+      <body>
+      <h1>Touchcreator</h1>
       <img src="Touch.png">
+<p>This is my website. So touchcreator.tk is old now. This also means that touchcreator.tk will now redirect here. this site is far from finished.</p>
+      <p><a href=https://github.com/Touchcreator>GitHub</a>
+        <a href=https://scratch.mit.edu/users/Touchcreator>Scratch</a>
+        <a href=https://www.reddit.com/u/Touchcreator>Reddit</a>
+        <a href=https://open.spotify.com/user/uwbirozgcs5bj7qs36nw48xye>Spotify</a>
+        <a href=https://steamcommunity.com/profiles/76561199091910255>Steam</a>
+        <a href=https://www.twitch.tv/touchcreator>Twitch</a>
+        <a href=https://twitter.com/Touchcreator1>Twitter</a>
+        <a href=https://www.youtube.com/channel/UCDIknHfpv3G5OCpQFb-lYVQ>YouTube</a>
```

Comment:
I used all of the links on @Touchcreator's Discord profile and added a link to their Scratch profile.